### DISCUSSION TITLE
Watermark Option

### DISCUSSION THREAD

**Discussion by tikilab** [acheong08/Diffusion-ColabUI#123] [General]
I noticed "Do not add watermark to images" has been removed from the latest version. Was this intentional or an oversight? What is/was the practical use for this setting?

**Comment by acheong08** [comment]
I have no idea what that option was. I never got any watermarks.

**Reply by acheong08** [reply]
I think they used to have a hidden watermark in images to fingerprint them as AI generated. @automatic1111 abandoned his project so I swapped to https://github.com/vladmandic/automatic which may have different opinions on that matter.
